packages/mds-json-schema/exceptions.ts

```diff
@@ -0,0 +1,8 @@
+import { BaseError, reason } from '@mds-core/mds-utils'
+
+/* istanbul ignore next */
+export class ValidationError extends BaseError {
+  public constructor(error?: Error | string, public info?: unknown) {
+    super('ValidationError', reason(error))
+  }
+}
```

Comment:
I would just leave this in `mds-utils` and import it from there. `ValidationError` is potentially useful in other contexts.

Reply:
Would it make sense to move the exception definitions to ids-types?

Reply:
Not opposed.

Reply:
Actually I think this would be boiling the ocean for very little gain so I'm going to hold off.